apps/website-25/src/components/Nav.stories.tsx

```diff
@@ -39,8 +39,5 @@ export const Default: Story = {
 export const Customized: Story = {
   args: {
     logo: imgSrc,
-    children: [
-      <><a href="#">Support Us</a><a href="#">About</a><a href="#">Join us</a><a href="#">Blog</a></>,
```

Comment:
This was already unused since refactoring the links into the Nav component itself